Buildings/Fluid/HeatExchangers/Examples/WetCoilCounterFlowPIControlAutoTuning.mo

```diff
@@ -9,7 +9,7 @@ model WetCoilCounterFlowPIControlAutoTuning
   Buildings.Controls.OBC.Utilities.PIDWithAutotuning.FirstOrderAMIGO
```

Comment:
In the plot results, can we add two more diagrams to show the `con.triTun` and `con.y`, so explicitly show when the autotuning begins and how the control changes? Like below:
<img width="1116" height="941" alt="Screenshot 2025-10-07 at 3 11 33 PM" src="https://github.com/user-attachments/assets/d4cd192c-a842-466f-b234-8b1451841af7" />